README.md

```diff
@@ -19,14 +19,11 @@ build static site
 yarn run deploy
 ```
 
-Cool things:
+note for banners
+
+If a banner is intended to be script-free, the relevant tags
+(`<link>`, `<script>`) need to be removed manually, and moved to `static_html/`
+which will be automatically copied on `export`. There is also a sub-command
+`copy-static` available, which allows you to do this step without re-running
+`export`.
 
-- [x] prefetch links & code splitting (performance of a website with app responsiveness there after)
```

Comment:
That was requested by @craig-mulligan, the only remaining to be done live as issues now.